### PR TITLE
[fix #7132] New Firefox accounts page

### LINKED ISSUE #7132: Redesign /accounts page for Trailhead
## Description

Redesign the /accounts page with content for Trailhead launch on June 4

[Master project doc](https://docs.google.com/document/d/13CqvenWPOc8MQpPFIROjoSumq2RldehhIeYIjmpDi9w/edit)

[Copy doc](https://docs.google.com/document/d/14yH7BAC2qN5LKq3FsZITJS21moCFQZGgstx-014Oeyw/edit?usp=sharing)

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] UI designs are received by 5/7
- [x] Interaction designs per user state are received by 5/7
- [x] Copy is received by 5/10 (was 5/15)
- [x] Strings are extracted by 5/21 (earlier better)

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [ ] Code page w. delivered copy and design assets
- [ ] Analytics (@sghoseWI) review

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Redesigns the accounts benefits page at /firefox/accounts/

## Issue / Bugzilla link
#7132 

## Testing
https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/accounts/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/accounts-2019.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  <div class="c-accounts-value-container">
+    <div class="c-accounts-value-media">
+      {{ high_res_img('firefox/accounts/trailhead/value-respect.jpg', {'alt': '', 'width': '800', 'height': '600'}) }}
+    </div>
+
+    <div class="c-accounts-value-content">
+      <h3 class="c-accounts-value-title">{{ _('Get the respect you deserve.') }}</h3>
+
+      <p>
+      {% trans promise='https://blog.mozilla.org/firefox/firefox-data-privacy-promise/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=accounts-trailhead&amp;utm_content=accounts-value&amp;utm_term=respect-you-deserve' %}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/accounts-2019.html]: Note: this blog post is not yet live, and blocks this page from going to production. I'll consult with MJ on when the post can be published.

File: media/js/firefox/accounts-2019.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
```
Comment (stephaniehobson) [inline media/js/firefox/accounts-2019.js]: Cleanup: empty file can go.

File: media/css/firefox/accounts-2019.scss [hunk 3]
```diff
@@ -0,0 +1,564 @@
...
+}
+
+.c-accounts-movement-container {
+    margin: $layout-xl auto;
+
+    @media #{$mq-md} {
+        @supports (display: grid) {
+            display: grid;
+            grid-template-columns: repeat(2, 1fr);
+            grid-template-rows: 2;
```
[the remark this answers is not in the document]
Comment (craigcook) [reply media/css/firefox/accounts-2019.scss]: doh, just an artifact.

File: media/js/firefox/accounts-2019.js (hunk 2 above)
Comment (craigcook) [reply media/js/firefox/accounts-2019.js]: Yeah, I ended up not even needing any special JS. Forgot to clean this up (and the bundle).

**Comment by craigcook** [comment]
I'm going to un-draft this because I think it's done, but it's still Do Not Merge pending some info and/or guidance on the blog post link.

File: media/css/firefox/accounts-2019.scss [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+    width: 100px;
+
+    span {
+        display: block;
+        font-weight: normal;
+    }
+
+    .t-product-firefox & {
+        background-image: url('/media/img/logos/firefox/logo-quantum.png');
+        @include background-size(50px, 50px);
```
[the remark this answers is not in the document]
Comment (craigcook) [reply media/css/firefox/accounts-2019.scss]: Another artifact I needed to clean up. Originally they were all slightly different sizes but that annoyed me so I made them all perfectly square, just hadn't tidied this up yet.

File: bedrock/firefox/templates/firefox/accounts-2019.html [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+      <h1 class="c-accounts-hero-title">{{ _('There is a way to protect your privacy. <span>Join Firefox.</span>') }}</h1>
+
+      {# L10n: This refers to the way some tech companies treat personal information as a commodity, as if their users are the product they're selling to their advertisers. #}
+      <p class="c-accounts-hero-desc">{{ _('Take your stand against an industry that’s making you the product.') }}</p>
+    </div>
+
+    <div class="c-accounts-form">
+      {{ fxa_email_form(
+        entrypoint='mozilla.org-accounts_page',
+        all_browsers=True,
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/accounts-2019.html]: Does this mean we're no longer signing Firefox users into Sync as well?

Comment (craigcook) [reply bedrock/firefox/templates/firefox/accounts-2019.html]: oh... damn, you're right. Hmm... ok, need to rethink this.

**Comment by craigcook** [comment]
This page is now behind the switch `firefox_accounts_trailhead` so we can push it to production ahead of the blog post, then activate it when we're ready.

**Comment by stephaniehobson** [comment]
R+ for the switch and form parameters.

The "Already have an account?" link also has the sync parameter on it, I suppose we have to deal with that too?

**Comment by stephaniehobson** [comment]
I'm okay merging and tiding it after.

**Pull request merged**